lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -469,6 +470,16 @@ spec = do
             , "  EPOCH_NUMBER             epoch number parameter or 'latest'"
             ]
 
+        ["network", "clock", "--help"] `shouldShowUsage`
+            [ "Usage:  network clock [--port INT]"
+            , "  View NTP offset."
```

Comment:
_Maybe_ the description could be more elaborate such that users that might not even know what NTP is can understand what this command is useful for.

But there's a neatness to this conciseness too.